packages/component/src/Composer.js

```diff
@@ -212,6 +219,20 @@ const Composer = ({
     };
   }, [dispatch, directLine, userID, username]);
 
+  const internalMarkdownIt = useMemo(() => new MarkdownIt(), []);
+
+  const internalRenderMarkdownInline = useMemo(
```

Comment:
Do we need to sanitize the resulting HTML? Also do we want the `internalMarkdownIt` to respect the carriage return style option?

https://github.com/microsoft/BotFramework-WebChat/blob/master/packages/bundle/src/renderMarkdown.js